libraries/eosiolib/contracts/include/eosio/action.hpp

```diff
@@ -54,6 +54,28 @@ namespace eosio
       }
    };  // namespace internal_use_do_not_use
 
+   template <typename T>
+   struct excluded_arg
```

Comment:
The name `excluded_arg` is confusing especially when combined with `ignore` which sounds vaguely similar but means almost the opposite.